### DISCUSSION TITLE
Beginner guide?

### DISCUSSION THREAD

**Discussion by hw-monkey** [Kaldek/rat-ratgdo#117] [Q&A]
I have soldered and performed troubleshooting on circuit boards, even repairing boards impacted from lightning strikes. I have followed guides to take assemble esphome devices. But I have never printed a board, ordered a pre-assembled board, etc.

Is there a simple guide for helping a person decide whether they should use a ESP32 vs ESP8266, how to figure out the circuit board size needed for PCBWAY, and what to upload to a service like that?

I know this sound SUPER newbie, and it is. But I think that Wieland's prices now are ridiculous (basically $70 incl shipping) and I am sure others are thinking the same. All I need is a little help to get me started and I think this could be a fun project.

Thank you

**Comment by FireflyJar** [comment]
I saw this question asked before and someone answered 'the ESP32 is an overkill'
I am going with ESP8266 for now because it's what I have kicking around.
